migrations/add_audit_count_columns_mysql.py

```
@@ -0,0 +1,79 @@
+
+from utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+def column_exists(engine, table_name: str, column_name: str) -> bool:
+    """Check if a column exists in a table"""
+    inspector = inspect(engine)
+    try:
+        columns = [col['name'] for col in inspector.get_columns(table_name)]
+        return column_name in columns
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)